items/item-history.js

```diff
@@ -1,7 +1,5 @@
-// const { ZonedDateTime } = require('@js-joda/core');
-const time = require('../time');
+const time = require('@js-joda/core');
```

Comment:
out of curiosity, whats the benefit of not using the internal time class that also has this import ? (so now there are multiple places importing an external lib)

Reply:
When importing the internal time library, I was not able to get the type definitions for it and then the functions that return a ZonedDateTime had no type definitions.

Reply:
You can import `ZonedDateTime` from @js-joda/core via a `@typedef` statement in the docs. Then the `require` can point to our own `time.js`:

```js
const time = require('../time');

/**
 * @typedef { import('@js-joda/core').ZonedDateTime } ZonedDateTime
 */
```

In the JSDoc comments `ZonedDateTime` can be referenced directly, e. g.: 

```js
  /**
   * Gets the average value of the state of a given Item between two certain points in time.
   *
   * @param {(ZonedDateTime | Date)} begin begin
   * @param {(ZonedDateTime | Date)} end end
   * @param {string} [serviceId] Optional persistence service ID, if omitted, the default persistence service will be used.
   * @returns {(number | null)}
   */
  averageBetween (begin, end, serviceId) {
    return this._decimalOrNull(PersistenceExtensions.averageBetween(this.rawItem, ...arguments));
  }
```

Reply:
@sfriedle Thanks for the suggestion, I‘ve already tried this a few times and it always caused probems with the JSDoc generation, but luckily I found a JSDoc plugin that solves that issue.